Assets/10_Miscellaneous/03_Settings/CFG_Game Settings.asset

```diff
@@ -55,7 +56,7 @@ MonoBehaviour:
   _autoMap: 0
   _autoMapClearsConsole: 0
   _levels:
-  - {fileID: 11400000, guid: 7b7aac6db6bc67d4180797a86dcde916, type: 2}
+  - {fileID: 0}
   - {fileID: 11400000, guid: 2025cb8784e11194e81a22be4121a4d9, type: 2}
   - {fileID: 11400000, guid: 8c9650dfd73baa2488815b6d49277ad2, type: 2}
   - {fileID: 11400000, guid: 4d39162ec980a0248bab8aed672be0c0, type: 2}
```

Comment:
You did not test the game. That's obvious, because your Level 08 is not added to the Levels.

-> Test the game before you push

Reply:
No idea what are you seeing, it looks fine on my end
![image_2024-08-06_151133896](https://github.com/user-attachments/assets/07fcfa47-590b-466e-a7f2-0d7844e2e45c)

Reply:
These are the changes on the server. Also what I downloaded from the server.

Verify that you uploaded your work properly.

Reply:
Sadly not. You just uploaded a font.

Reply:
Now they are here.